docs/application-connector/03-08-application-gateway-proxying-requests.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Application Gateway
+type: Details
+---
+
+>**CAUTION:** This document describes the proxy service in the alternative **gatewayOncePerNamespace** [mode](#architecture-application-connector-components-application-operator). To read about the proxy service in the default legacy mode, see [this](#architecture-application-gateway) document. 
```

Comment:
```suggestion
>**CAUTION:** This document describes the proxy service in the alternative [**gatewayOncePerNamespace** mode](#architecture-application-connector-components-application-operator). To read about the proxy service in the default legacy mode, see [this](#architecture-application-gateway) document. 
```

Reply:
See [this](https://github.com/kyma-project/kyma/pull/7784#discussion_r397880117)